src/openms/include/OpenMS/CHEMISTRY/EnzymaticDigestion.h

```diff
@@ -113,7 +113,7 @@ namespace OpenMS
     Size peptideCount(const AASequence& protein);
 
     /// Returns true if peptide at position @p pep_pos with length @p pep_length within protein @p protein was generated by the current model
-    bool isValidProduct(const AASequence& protein, Size pep_pos, Size pep_length, bool methionine_cleavage = false);
+    bool isValidProduct(const AASequence& protein, Size pep_pos, Size pep_length, bool methionine_cleavage = false, bool ignore_missed_cleavages = true) const;
```

Comment:
can you change the `src/pyOpenMS/pxds/EnzymaticDigestion.pxd` signature accordingly to 

```
bool isValidProduct(AASequence protein, Size pep_pos, Size pep_length, bool methionine_cleavage, bool ignore_missed_cleavages) nogil except +
```

thanks